css-showcases/src/z-index-helper.js

```diff
@@ -26,7 +25,7 @@ export const getZIndexHtml = (props) => {
     .join(' ');
 
   return /*html*/ `
-    <div style="display:flex;flex-direction:row;justify-content:center;">
+    <div style="display:flex;flex-direction:row;justify-content:center;margin-left:9rem;">
```

Comment:
be aware that won't be responsive.

Reply:
But at least should better than before 👍

Reply:
👍